packages/transfer-app/src/Saved.tsx

```diff
@@ -0,0 +1,155 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { KeyringPair } from '@polkadot/keyring/types';
+import { ApiContext } from '@polkadot/ui-api';
+import { AddressSummary, Grid, MarginTop, NavLink, Stacked, StyledLinkButton, SubHeader, WalletCard, WithSpace } from '@polkadot/ui-components';
+import { KeyringAddress } from '@polkadot/ui-keyring/types';
+
+import React from 'react';
+import { Link, RouteComponentProps } from 'react-router-dom';
+
+interface MatchParams {
+  currentAddress: string;
+}
+
+interface Props extends RouteComponentProps<MatchParams> {
+  basePath: string;
+  onSelectAddress: (address: string) => void;
+}
+
+type State = {
+  allAccounts: Array<KeyringPair>,
+  allAddresses: Array<KeyringAddress>
+};
+
+type Memo = {
+  [index: string]: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+};
+
+export class Saved extends React.PureComponent<Props, State> {
+  static contextType = ApiContext;
+
+  context!: React.ContextType<typeof ApiContext>; // http://bit.ly/typescript-and-react-context
+
+  state: State = {
+    allAccounts: [],
+    allAddresses: []
+  };
+
+  memo: Memo = {};
+
+  componentDidMount () {
+    const { keyring } = this.context;
+
+    this.setState({
+      allAccounts: keyring.getPairs(),
```

Comment:
I believe there's a way to use `ui-keyring/observables/accounts` for this, so that we don't need to setState. Still not 100% sure, so we can leave it like this for now.

Reply:
@amaurymartiny I added an example of that to the ui-keyring docs https://github.com/polkadot-js/ui/tree/master/packages/ui-keyring, I'll make a commit using the observable